testdata/project-v2/main.go

```diff
@@ -72,7 +74,7 @@ func main() {
 		setupLog.Error(err, "unable to create controller", "controller", "Captain")
 		os.Exit(1)
 	}
-	if err = (&crewv1.Captain{}).SetupWebhookWithManager(mgr); err != nil {
+	if err = (&v1.Captain{}).SetupWebhookWithManager(mgr); err != nil {
```

Comment:
Same issue here.

Reply:
Solved here: https://github.com/kubernetes-sigs/kubebuilder/pull/1073 and cherry-picked in this PR.